build-logic/performance-testing/src/main/groovy/gradlebuild/performance/tasks/RebaselinePerformanceTests.groovy

```diff
@@ -22,8 +22,10 @@ import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.SourceTask
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.options.Option
+import org.gradle.work.DisableCachingByDefault
 
 @CompileStatic
+@DisableCachingByDefault(because = "Not worth caching")
 abstract class RebaselinePerformanceTests extends SourceTask {
```

Comment:
This also has no cacheable output, because of the in-place modification, no?

Reply:
Yes.